docs/guide/learning/learning-platforms.md

```diff
@@ -32,46 +32,40 @@ The instructor and author of the platform, Jeffrey Way, is a pro when it comes t
+[Frontend Masters](https://frontendmasters.com/) is an online video learning platform for JavaScript and front-end development, covering topics such as React, Node.js, Vue, Functional JavaScript, CSS Grid, Flexbox and more!
 
-Their videos are top notched and goes in-depth in the subject topic.
+The videos have top notch quality and go in-depth for the specific topic. If you are just starting out, there is a great [Introduction to Vue.js](https://frontendmasters.com/courses/vue/) that will bring you up to speed in no time.
 
-Their instructors are industry leaders and even the creator of Vue has a course also. Definitely worth checking out!
+Their instructors are industry leaders, with the creator of Vue himself, Evan You, leading a [course](https://frontendmasters.com/courses/advanced-vue/) there. Definitely worth checking out!
 
 ### Egghead.io
-[Egghead.io](https://egghead.io/) is a group of working web development professionals and open source contributors that provide you with concise, information dense video courses on the best tools in the industry.
+[Egghead.io](https://egghead.io/) offers high quality content by a group of working web development professionals and open source contributors. They provide concise and dense video courses on some of the best tools in the industry.
 
-Their videos are straight to the point and deliver knowledge that you can use today. You won't find boring 8 hour courses on egghead, instead you'll find a wealth of bite-sized courses that pack more information in a fraction of the time.
+Their videos are straight to the point and deliver knowledge that you can use today. If you don't have time for long hours of tutorials, Egghead offers a wealth of bite-sized courses that pack more information in a fraction of the time.
 
-Egghead instructors are industry veterans with years of practical,
-from-the-trenches experience building real software at all scales.
+Egghead instructors are industry veterans with years of practical, from-the-trenches experience building real software at all scales.
 
 ### Codecourse
 [Codecourse](https://codecourse.com) is a learning platform whose mission is to help you to learn to code with relaxed, easy to follow tutorials.
 
-Their videos are great and focuses on practical projects to get you ahead. Rather than just teach theory, they are focused on real code to inspire your next project. For Laravel Developers that want to learn Vue, this is a great place to start!
-
-The instructor is great and definitely understands the topics really well.
-
-
-
+Their videos are great and focus on practical projects to get you ahead. Rather than just teach theory, they are focused on real code to inspire your next project. For Laravel Developers that want to learn Vue, this is a great place to start! 
```

Comment:
Same as above. We should showcase at least a course or two? Maybe mention they have paths that you can follow?